pages/apis/rest_api/pipeline_templates.md

```diff
@@ -0,0 +1,293 @@
+# Pipeline templates API
```

Comment:
@jonathanly my expectation when loading up the docs was that this would be under "Pipelines REST API > Templates" or "Pipelines REST API > Pipeline Templates".

Is there a particular reason why this warrants a top-level section?

Reply:
On looking at it again, it doesn't make sense at all so I've moved the page under Pipelines REST API ([e7d11cb](https://github.com/buildkite/docs/pull/2479/commits/e7d11cb314de0cdb038a225b783d28bc4dc7e420))